Alcoholic Additives

Sylrene opened the pull request:
Adds new mixed drinks and options to the alcohol dispensers.

Sylrene committed:
Alcoholic Additives

Adds new mixed drinks and options to the alcohol dispensers.

io-util commented on code/modules/reagents/dispenser/dispenser_presets.dm:
```diff
@@ -118,6 +118,10 @@
 			/obj/item/weapon/reagent_containers/chem_disp_cartridge/cognac,
 			/obj/item/weapon/reagent_containers/chem_disp_cartridge/ale,
 			/obj/item/weapon/reagent_containers/chem_disp_cartridge/mead
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/mead,
```
This should not be here 2 times, the first is missing a comma too. Remove one of them and make sure the one that stays has a comma, else the list will break.

Sylrene committed:
Update dispenser_presets.dm

Fixing a segment of code where a line existed twice and incorrectly functioned anyways.

EvenInDeathIStillServe commented on code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm:
```diff
@@ -3002,3 +3002,142 @@
+
+/datum/reagent/ethanol/creme_de_menthe
+	name = "Creme de Menthe"
+	description = "A sweet, mint-flavored alcoholic beverage"
```
```suggestion
	description = "A sweet, mint-flavored alcoholic beverage."
```

EvenInDeathIStillServe commented on code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm:
```diff
@@ -3002,3 +3002,142 @@
+		return
+	if(alien == IS_SKRELL)
+		strength_mod *= 2
+	if(prob(1*strength_mod))
```
```suggestion
	if(prob(strength_mod))
```
1*something = something

EvenInDeathIStillServe commented on code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm:
```diff
@@ -3002,3 +3002,142 @@
+	strength = 13
+
+	glass_name = "Stinger"
+	glass_desc = "Holding this glass will make you seem like high-soceity... if you were from the 1800s."
```
```suggestion
	glass_desc = "Holding this glass will make you seem like high-society... if you were from the 1800s."
```

Mojamble replied:
we live in a soceity

Also, I should have gone over this myself before handing it over to Sy. This is on me.

EvenInDeathIStillServe reviewed (changes requested):
Just a couple grammar mistakes, and an unnecessary multiplication.

io-util reviewed (changes requested):
Fix the small stuff suggested above and it's a go from me. @Sylrene

Sylrene committed:
Update Chemistry-Reagents-Food-Drinks.dm

Fixed some lines of code which had grammatical errors and unnecessary additions.

io-util reviewed (approved):
Looks okay now.
